script/lib/azput.js

```diff
@@ -7,13 +6,13 @@ const blobServiceClient = BlobServiceClient.fromConnectionString(process.env.ELE
 
 const args = require('minimist')(process.argv.slice(2));
 
-let { prefix = '/', key_prefix = '', _: files } = args;
+let { prefix = '/', keyPrefix = '', _: files } = args;
```

Comment:
Wouldn't this be a functional change? I assume `minimist` is turning an option like `--key-prefix` into `key_prefix`.

Reply:
good point, reverting